CHANGELOG.md

```diff
@@ -1,5 +1,101 @@
 # Change Log
 
+## [0.9.0](https://github.com/confluentinc/ksql/releases/tag/v0.9.0-ksqldb) (2020-05-08)
+
+### Features
+
+* add multi-join expression support ([#5081](https://github.com/confluentinc/ksql/pull/5081)) ([002cd5a](https://github.com/confluentinc/ksql/commit/002cd5ad6473cbb000291e32ebe5a459d2870b61))
+* support more advanced suite of LIKE expressions ([#5013](https://github.com/confluentinc/ksql/pull/5013)) ([67cd9d9](https://github.com/confluentinc/ksql/commit/67cd9d91690039cc01f43438db1fbcd3fe4924ff))
+* add COALESCE function ([#4829](https://github.com/confluentinc/ksql/pull/4829)) ([251c237](https://github.com/confluentinc/ksql/commit/251c237c455f4cf3843628f41e4bbf57795cfd12))
+* add internal request to KsqlRequestConfig and SessionProperties ([#4771](https://github.com/confluentinc/ksql/pull/4771)) ([fc10cae](https://github.com/confluentinc/ksql/commit/fc10caeb6c164febd8041bc523e765fa6a6b77d8))
```

Comment:
This can be removed this from the features list.

Reply:
Understand now. 
I removed the rest of the features.

Reply:
I don't understand. Why did we remove a bunch of features from the list: ksqlquerystatus, muti-way joins, pull query rate limiting, klip 14, scatter gather, and others?

Reply:
I put them back. I misunderstood the `this` with `these`, so removed everything below. I also removed some the lines related to vert.x changes as they're not new features.